rttest/test/test_api.cpp

```diff
@@ -48,13 +48,13 @@ TEST(TestApi, read_args_get_params) {
   struct rttest_params params;
   EXPECT_EQ(0, rttest_get_params(&params));
 
-  EXPECT_EQ(params.iterations, 4321);
+  EXPECT_EQ(params.iterations, static_cast<uint>(4321));
```

Comment:
There's a nice trick to reduce the noisiness of the casts.  You can do:

```suggestion
  EXPECT_EQ(params.iterations, 4321u);
```

(see https://www.geeksforgeeks.org/integer-literal-in-c-cpp-prefixes-suffixes/ for more).  Same goes below.